ha/const.py

```diff
@@ -141,6 +142,8 @@
 ALERT_FILTER_INDEX = "alert_filter_rules"
 ALERT_FILTER_RULES_FILE = "{}/alert_filter_rules.json".format(CONFIG_DIR)
 SOURCE_ALERT_FILTER_RULES_FILE = "{}/alert_filter_rules.json".format(SOURCE_CONFIG_PATH)
+SYSTEM_SERVICE_FILE = "{}/event_analyzer.service".format(SYSTEM_DIR)
```

Comment:
I am not sure but is using '-' instead of '_' in service name preferred?

Reply:
I could find no such convention. 
SSPL writes it using "-" but csm is using a "_"
So to keep names consistent  between the daemon file and service file I went with "_"

Reply:
ok